jobmatch/src/main/resources/data.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Seed the Role database with possible user roles
+--
+MERGE INTO ROLE AS t USING (VALUES(1,'user'), (2, 'company')) AS vals(a,b)
```

Comment:
Gotcha. For initial seeding, those constraints should never be relevant. There's often a command to temporarily turn off constraints like that, but I agree we wouldnt want to break the relations.